### ISSUE TITLE
Incorrect key is returned in SettingsManager->all()  

### ISSUE THREAD

**Issue by m-safwan** [anlutro/laravel-settings#165]
Incorrect key is returned when using ```SettingsManager->all()``` if the keys are stored as numeric values such as ```1234```, ```0``` etc.

Code:
```
        $laravelSettings = new SettingsManager(App::getFacadeRoot());

        $laravelSettings->setExtraColumns(['entity_id' => 'test1', 'entity_type' => 'testing', 'module' => 'test',
            'created_at'  => Carbon::now()->getTimestamp(), 'updated_at'  => Carbon::now()->getTimestamp()
        ]);

        $laravelSettings->set('1234', 'abcd');

        $laravelSettings->set('15', 'pqrs');

        $laravelSettings->save();

        $value1 = $laravelSettings->get('1234');

        $value2 = $laravelSettings->get('15');

        $laravelSettings = new SettingsManager(App::getFacadeRoot());

        $value3 = $laravelSettings->all();

        print_r($value1);

        print_r($value2);

        print_r($value3);
```

Output:
```
abcd
pqrs
Array
(
    [0] => abcd
    [1] => pqrs
)
```

Analysis:
This issue is due to this [line](https://github.com/anlutro/laravel-settings/blob/master/src/SettingStore.php#L234) of code in SettingsStore.php:232. This line uses ```array_merge``` to merge the two arrays. If the arrays contain numeric strings as keys (such as '1234', '15', etc'), then ```array_merge``` will renumber the indexes (starting with 0) in the resultant array. 
Reference: https://www.php.net/manual/en/function.array-merge.php

Fix:
We can use the ```+``` operator to correctly merge the two arrays.

**Comment by m-safwan** [comment]
@anlutro Could you please check this?

**Comment by bweston92** [comment]
@m-safwan if you create a pull request with a fix and test's to demonstrate it is now working I would merge and do a new major release.

**Comment by m-safwan** [comment]
@bweston92 Sure, I've created https://github.com/anlutro/laravel-settings/pull/166 to fix this.

**Comment by bweston92** [comment]
Thanks @m-safwan for providing fix and test case, please upgrade to `v1.2.0`
